.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I'll pluck it out. Eventually we should create helpful plugin debugging for ESLint. Ideally something better than using http://astexplorer.net/